lib/taskana-core/src/main/java/pro/taskana/monitor/internal/utils/ReportsHelper.java

```diff
@@ -0,0 +1,14 @@
+package pro.taskana.monitor.internal.utils;
+
+public class ReportsHelper {
+    public static String[] toLowerCopy(String... source) {
```

Comment:
Please remove this file and use `BaseQuery#toLowerCopy` instead. For this you need to make `BaseQuery#toLowerCopy` static and public ;)

Reply:
Thanks @mustaphazorgati. Updated the PR following the 1st suggestion. 
Not using `google-java-format` plugin. Unsure what went wrong there tbh. 
Will monitor and update if build fails again.